app/renderer/components/styles/commonStyles.js

```diff
@@ -198,6 +198,21 @@ const styles = StyleSheet.create({
   isCommonForm: {
     fontSize: globalStyles.fontSize.flyoutDialog,
     width: '100%'
+  },
+
+  linkText: {
+    display: 'inline-block',
+    textDecoration: 'underline',
+    color: globalStyles.color.braveOrange,
+    cursor: 'pointer',
+    margin: '0',
+    ':hover': {
+      color: '#000'
+    }
+  },
+
+  linkTextSmall: {
```

Comment:
@evq would you please change `linkTextSmall` into `linkText_small` based on our guideline, which is available here? https://github.com/brave/browser-laptop/blob/master/docs/style.md#defining-our-blocks-elements-and-modifiers thanks!